### PR TITLE
Disable analysis

### PR DESCRIPTION
**Pull request by pylaligand**

### CODE REVIEW

File: examples/hello_flutter/lib/main.dart [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 import 'dart:ui' as ui;
 
-void beginFrame(Duration timeStamp) {
+void _beginFrame(Duration timeStamp) {
```
Comment (Hixie) [inline examples/hello_flutter/lib/main.dart]: why this change?

Comment (pylaligand) [reply examples/hello_flutter/lib/main.dart]: Whoops I didn't see that one sneak in. I think I had to do that because it raised an error that a public member was not properly documented. Consistently with my other change, I think in Fuchsia we should disable analysis on this package too.

Comment (pylaligand) [reply examples/hello_flutter/lib/main.dart]: Updated the change to only affect build files.
